requirements.txt

```diff
@@ -8,11 +8,10 @@ colorama
 termcolor
 click
 asciitree
-fastapi
+fastapi[standard]
 weave
 agentops
 langchain
 langchain_core
 watchdog
-agentops
```

Comment:
Obviously I'm biased, but users are getting a lot of value out of this package. LlamaFS isn't actively maintained, and this is a valuable debugging package to keep in

Reply:
This is a duplicate line. I got an agentops account and it is actually really useful!

You might need to amend `fastapi` over to `fastapi[standard]`. I wasn't able to use `fastapi dev server.py` until I installed this specific version.